internal/network/machine_network_cidr.go

```diff
@@ -417,6 +450,57 @@ func GetInventoryNetworksByFamily(hosts []*models.Host, log logrus.FieldLogger)
 	return ret, nil
 }
 
+type CidrMetric struct {
+	cidr   string
+	metric int32
+}
+
+func GetDefaultRouteNetworkByFamily(hosts []*models.Host, log logrus.FieldLogger) map[AddressFamily]string {
```

Comment:
Should not it be only for sno? why to pass hosts?

Reply:
fixed